README.md

```diff
@@ -67,16 +67,17 @@ ObjectMapper mapper = new ObjectMapper()
 
 ### Serialization
 
-For serialization this module currently supports the following data types:
-
-| Input                                                                                                                             | Standard                                          | Output                                 |
-|-----------------------------------------------------------------------------------------------------------------------------------|---------------------------------------------------|----------------------------------------|
-| [`javax.money.CurrencyUnit`](https://github.com/JavaMoney/jsr354-api/blob/master/src/main/java/javax/money/CurrencyUnit.java)     | [ISO-4217](http://en.wikipedia.org/wiki/ISO_4217) | `EUR`                                  |
```

Comment:
Not publicly supporting `CurrencyUnit` and `NumberValue` anymore is technically a breaking change, but since we're preparing 2.x anyway, that's OK. It also gives us some more freedom in the future.